R/readUNFCCC_NDC.R

```diff
@@ -1,107 +1,114 @@
 #' Reads NDC policy database with capacity, emission, and share targets, originally based on Rogelj et al. 2017
+#'
 #' @description Reads excel sheet with NDC (Nationally Determined Contributions)
-#' data on different policy targets (capacity, emission, and share targets) with different variations
-#' @details Country name is ISO coded. Capacity/Additional Capacity targets are in GW. Generation/Production targets are in GWh.
+#'  data on different policy targets (capacity, emission, and share targets) with different variations
+#'
+#' @details Country name is ISO coded. Capacity/Additional Capacity targets are in GW. Generation/Production targets
+#'  are in GWh.
 #' @return  magpie object
 #' @author  Aman Malik, Christoph Bertram, Oliver Richters, Sophie Fuchs, Rahel Mandaroux
 #' @param subtype Capacity_2023_cond (or 2018/2021/2022 or uncond) for capacity target,
-#' Emissions_2023_cond (or 2018/2021/2022 or uncond) for Emissions targets
-#' @importFrom readxl read_xlsx
-#' @importFrom dplyr select
+#'  Emissions_2023_cond (or 2018/2021/2022 or uncond) for Emissions targets
+#' @param subset A string (or vector of strings) designating the scenario(s) to be returned.
```

Comment:
Maybe add a remark that it is needed for the convert function, as it is not used directly in the read function

Reply:
OK!